src/util/apiUtils.js

```diff
@@ -270,10 +275,13 @@ export function parseRequestHeaders(request) {
 export function parseRequestQueries(request) {
 	const {
 		method,
+		mime,
```

Comment:
So I guess my real question is how are we going to set the content-type in `mup-web`?